src/components/CCPDashboard/PatientOverview.tsx

```diff
@@ -0,0 +1,266 @@
+import React from "react";
+import clsx from "clsx";
+import { Colours } from "../../styles/Constants";
+import {
+  Box,
+  Typography,
+  makeStyles,
+  Card,
+  Grid,
+  TableContainer,
+  Table,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+} from "@material-ui/core";
+import { useQuery } from "@apollo/react-hooks";
+import {
+  FETCH_ALL_PATIENTS,
+  Patient,
+  TriageLevel,
+  Status,
+} from "../../graphql/queries/templates/patients";
+import { TriageTag } from "./TriageTag";
+import { PatientInfoTable } from "./PatientInfoTable";
+import { ScanPatientButton } from "./ScanPatientButton";
+import LoadingState from "../common/LoadingState";
+
+interface PatientOverviewProps {
+  eventId: string;
+  ccpId: string;
+}
+
+interface TableRowData {
+  category: string;
+  count: number;
+  ratio: number;
+}
+
+const useStyles = makeStyles({
+  fullHeightGridItem: {
+    display: "flex",
+    alignSelf: "stretch",
+  },
+  card: {
+    padding: "24px",
+    marginTop: "16px",
+    marginRight: "24px",
+    height: "100%",
+  },
+  categoryTableCard: {
+    display: "flex",
+    alignItems: "center",
+    paddingRight: "44px",
+    paddingLeft: "44px",
+    marginTop: "16px",
+  },
+  lightBorder: {
+    borderColor: Colours.BackgroundGray,
+  },
+  noBorder: {
+    border: 0,
+  },
+  cellWithIcon: {
+    display: "flex",
+    alignItems: "center",
+  },
+  icon: {
+    marginRight: "9px",
+  },
+  patientTableCard: {
+    marginTop: "24px",
+    marginBottom: "145px",
+  },
+  scanButton: {
+    boxShadow: "0px 4px 10px rgba(0, 0, 0, 0.3)",
+    borderRadius: "2000px",
+    position: "fixed",
+    bottom: "56px",
+    right: "56px",
+    padding: "12px 26px",
+  },
+});
+
+export const PatientOverview = (props: PatientOverviewProps) => {
+  const classes = useStyles();
+
+  const { ccpId } = props;
+
+  // Should switch to fetching patients from cache
+  const { data, loading } = useQuery(FETCH_ALL_PATIENTS);
+  const allPatients: Array<Patient> = data ? data.patients : [];
+  const patients = allPatients.filter(
+    (patient: Patient) => patient.collectionPointId.id == parseInt(ccpId)
```

Comment:
using "===" breaks the code because of the types aren't equal :/ i tried parseInt(patient.collectionPointId.id) === parseInt(ccpId) but that didn't work either. I'll take another look at it but otherwise, not sure if it's worth spending more time on.